exercises/concept/elyses-enchantments/.docs/instructions.md

```diff
@@ -2,11 +2,11 @@
 
 As a magician-to-be, Elyse needs to practice some basics. She has a stack of cards that she wants to manipulate.
 
-To make things a bit easier she only uses the cards 1 to 10.
+To make things a bit easier she only uses the cards 1 to 10. Remember that array indices start at 0.
```

Comment:
This sentence feels a bit out of place, I would suggest the following alternative:
```suggestion
To make things a bit easier she only uses the cards 1 to 10 so her stack of cards can be represented by an array of numbers. The position of a certain card corresponds to the index in the array. That means position 0 refers to first card, position 1 to the second card etc.
```

Reply:
Thank you for the feedback. I have updated based on your suggestion.